File: auth-service/src/main/java/com/sudo248/authservice/repository/entity/Role.java
package com.sudo248.authservice.repository.entity;

public enum Role {
    CONSUMER("consumer"),
    STAFF("staff"),
    ADMIN("admin");

    private final String value;


    Role(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: client/SocStaff/app/src/main/java/com/sudo248/soc_staff/domain/entity/auth/Role.kt
package com.sudo248.soc.domain.entity.auth

enum class Role(val value: String) {
    CONSUMER("Thành viên"),
    STAFF("Nhân viên");

    companion object {
        fun fromStringValue(value: String): Role {
            if (value == "Nhân viên") return STAFF
            return CONSUMER
        }
    }
}